modules/plugins/ui/edgy-nvim/edgy-nvim.nix

```diff
@@ -0,0 +1,21 @@
+{lib, ...}: let
+  inherit (lib.options) mkOption mkEnableOption;
+  inherit (lib.types) bool;
+  inherit (lib.nvim.types) mkPluginSetupOption;
+in {
+  options.vim.ui.edgy-nvim = {
+    enable = mkEnableOption "edgy.nvim for predefined window layouts";
+    setRecommendedNeovimOpts = mkOption {
+      type = bool;
+      default = false;
+      description = ''
+        Whether nvf should set `vim.opt.laststatus` and `vim.opt.splitkeep` to
+        values recommended by upstream to ensure maximum compatibility.
+      '';
+    };
```

Comment:
You forgot to implement this?

Reply:
lol...

Reply:
Accidentally approved, meant to request changes.